app/src/main/java/vn/mbm/phimp/me/GalleryMap.java

```diff
@@ -214,15 +214,21 @@ public void run()
                 //show all photo in gallery
                 else {              
         		String[] projection = {MediaStore.Images.Media.DATA};
-        		Cursor cursor = managedQuery( MediaStore.Images.Media.EXTERNAL_CONTENT_URI,
+        		Cursor cursor = getContentResolver().query(
+						MediaStore.Images.Media.EXTERNAL_CONTENT_URI,
                         projection,
-                        null,null,
-                        null);
-        		int columnIndex = cursor.getColumnIndexOrThrow(MediaStore.Images.Media.DATA);
-        		for(int i=0; i<cursor.getCount(); i++){
-        			if(cursor.moveToNext())
-        			filepath.add(cursor.getString(columnIndex));
-        		}	
+                        null,
+						null,
+                        null
```

Comment:
@Abhi2424shek there was some tabs and space mismatch in my IDE and now I've fixed it and made a PR! :D